cypress/integration/InstancesTest.ts

```diff
@@ -13,27 +17,12 @@ describe("Instances Test", () => {
     });
 
     it("Submit", () => {
-      const newInstanceName: string = "Some new instance";
-      cy.ouiaId("create-smart-event-instance", "PF4/Button").click();
-      cy.ouiaId("create-instance", "PF4/ModalContent").then(($modal) => {
-        cy.wrap($modal)
-          .should("be.visible")
-          .within(() => {
-            cy.ouiaId("new-name", "PF4/TextInput").type(newInstanceName);
-            cy.ouiaId("info-instance-available-soon", "PF4/Alert").should(
-              "have.text",
-              "Info alert:Your Smart Events instance will be ready for use shortly after creation."
-            );
-            cy.ouiaId("submit", "PF4/Button").click();
-          });
-        cy.wrap($modal, { timeout: 20000 }).should("not.exist");
-      });
-
+      createInstance("Some new instance");
       cy.ouiaId("Instances list table", "PF4/Table")
-        .ouiaId(newInstanceName, "PF4/TableRow")
+        .ouiaId("Some new instance", "PF4/TableRow")
```

Comment:
@ChughShilpa, in case that the `newInstanceName` variable stay in code you cannot change this line.